lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -256,63 +256,60 @@ def add_level_two_and_three_data(xml, amount, data)
         xml.invoiceReferenceNumber data[:invoice_reference_number] if data.include?(:invoice_reference_number)
         xml.customerReference data[:customer_reference] if data.include?(:customer_reference)
         xml.cardAcceptorTaxId data[:card_acceptor_tax_id] if data.include?(:card_acceptor_tax_id)
-
         {
-          sales_tax: 'salesTax',
+          tax_amount: 'salesTax',
```

Comment:
Has this field been misnamed the whole time?

Reply:
So these fields made it into AM but never into core. With that in mind, I'm taking the time to normalize their approach a bit better so that it's similar to other gateways.